lib/os/CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ zephyr_sources(
   heap-validate.c
   bitarray.c
   multi_heap.c
+  memcpy32.c
```

Comment:
Let's have a Kconfig for this to not slow down all builds that don't use it.

Reply:
This doesn't seem necessary. When statically linking, only symbols that are used are kept. If `mcan` is not selected by Kconfig, then `memcpy32` doesn't end up in the zephyr binary.

Reply:
It's so that niche features that the majority don't use don't slow down the build time for everyone.

Reply:
I see